### DISCUSSION TITLE
A nested list component item hidden issue

### DISCUSSION THREAD

**Discussion by Slahser** [charmbracelet/bubbletea#443] [Q&A] [answered]
https://github.com/charmbracelet/bubbletea/pull/416/files

Run list-default main().

Here is a nested list structure,

list-default => press 'a' => into list-simple
list-simple => press 'h' => return to list-default

and items will be hidden. 

<img width="949" alt="image" src="https://user-images.githubusercontent.com/3760744/186629094-f82a88fb-ec1f-493c-b7ef-fe1966cc7a6c.png">

<img width="911" alt="image" src="https://user-images.githubusercontent.com/3760744/186629112-f6598468-bccf-4d99-8ae9-cf3c4f353d7f.png">

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/3760744/186629131-7a4e6e46-6694-43d1-a444-440ebc238c2a.png">

In another case, fancy list - fancy list perform like above too.  Even list => cmd => list .

**Comment by bashbunni** [comment, marked as answer]
Hi @Slahser,

The issue here is that the `tea.WindowSizeMsg` only gets sent on program start, so when you create a new default list, the list isn't being given a height and width which is why you can't see any items. Here is the diff for the changes needed to get this working:
[git diff](https://github.com/charmbracelet/bubbletea/files/9522059/diff.txt)

Or feel free to see the full files in [this gist](https://gist.github.com/bashbunni/2ef5da9acb042c1ae0732b1d682e5d9a)

**Reply by lemmebee** [reply]
I hated my life for sometime because of this! thanks @bashbunni
